### PR TITLE
[terra-list] Fix Items Views not truncating when used with chevrons in IE

### PR DESCRIPTION
**Pull request by Blackop778**
### Summary
Sets a width on the `Arrange` component used to display chevrons because `Arrange` uses flexbox and that's how IE likes it.

### Additional Details
Closes cerner/terra-clinical#243

### [Deployment](https://terra-core-deployed-pr-1592.herokuapp.com/#/tests/terra-list/list/selectable-list/item-view-truncation)

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
It looks like the item-display is rendering on top of the chevron

screen shot 2018-06-12 at 11 15 41 am

**Comment by Blackop778** [comment]
The overlap issue has been fixed
screen shot 2018-06-13 at 8 39 17 am

**Comment by dkasper-was-taken** [comment]
This can be solved easier. The issue with truncation on IE is that flex requires a set width to function. Arrange is an incomplete implementation, so you either need to replace the arrange implementation in the ListItem or easier set the fillAttributes.

```
<li {...customProps} role="option" aria-selected={isSelected} className={listItemClassNames} ref={refCallback}>
  <Arrange
    fill={content}
    fitEnd={chevron}
    align={'center'}
    fillAttributes={{ className: cx('content') }}
  />
</li>

**Comment by Blackop778** [comment]
Wow, that greatly simplifies things. Thanks!

File: packages/terra-list/package.json [hunk 1]
```diff
@@ -34,6 +34,9 @@
     "terra-doc-template": "^1.6.0",
     "terra-icon": "^2.13.0"
   },
```
Comment (emilyrohrbough) [inline packages/terra-list/package.json]: I think this needs to be moved to a dependency since we ship our test examples

**Comment by PaulSolDev** [comment]
Functional review complete.  No issues identified.

**Pull request merged**